solidity/contracts/GravityERC721.sol

```diff
@@ -0,0 +1,61 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "./Gravity.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { ERC721Holder } from "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "hardhat/console.sol";
+
+contract GravityERC721 is ERC721Holder, ReentrancyGuard, Ownable{
```

Comment:
I think we can take off the ownable tag now that we're doing the gravity.sol check another way.

Reply:
Removing...that should have been gone already.